virttest/qemu_vm.py

```diff
@@ -1897,6 +1900,48 @@ def sort_key(dev):
                               pci_bus=pci_bus)
                 iov += 1
 
+        def get_cid(cid):
+            """ Get an unused guest cid from system """
+            while cid:
+                cid_c = struct.pack('L', cid)
+                try:
+                    fcntl.ioctl(
+                        vsock_fd, arch.VHOST_VSOCK_SET_GUEST_CID, cid_c)
+                except IOError as e:
+                    if e.errno == errno.EADDRINUSE:
+                        cid += 1
+                        continue
+                    else:
+                        raise e
+                else:
+                    return cid
+
+        # Add vsock device, cid 0-2 are reserved by system
+        vsocks = params.objects('vsocks')
+        if vsocks:
+            vsock_path = "/dev/vhost-vsock"
+            if not os.path.exists(vsock_path):
+                logging.info("vsock module was not loaded, loading it...")
+                loaded = linux_modules.load_module('vsock')
+                loaded &= linux_modules.load_module('vhost_vsock')
```

Comment:
The module `vhost_vosck` depends on three modules: 
```
depends:        vmw_vsock_virtio_transport_common,vhost,vsock
```
Since modprobe will load all the dependent modules, we can remove `load_module('vsock')` here.

Reply:
@yanan-fu Updated, please help review, thanks.